documentation/src/main/asciidoc/topics/contributing/ui.adoc

```diff
@@ -0,0 +1,29 @@
+[[design_guidelines]]
+= UI Writing Guidelines
+
+This section describes guidelines for working with text in {brandname} user
+interface (UI) elements and applies, but is not limited, to the following
+components:
+
+* {brandname} Console
+* {brandname} CLI
+* log messages
+* error messages
+
+Content in the {brandname} UI should adhere to the link:https://www.patternfly.org/v4/ux-writing/about[PatternFly content style guide].
+
+While the PatternFly design guidelines are the definitive resource for UI writing guidelines, here are some rules for creating microcopy in {brandname} UI:
+
+* Headings on UI elements use sentence casing. link:https://www.patternfly.org/v4/ux-writing/capitalization/[PatternFly capitalization rules]
+* Dialogs that prompt for user action should have the following structure:
++
+Heading in question form, starting with verb. ("Create new entry?") +
+Body that provides details and outcome of action. +
+Body that uses a synonym to the verb in heading where ambiguity might exist. +
+Buttons.
+* Use past tense form for success or failure messages.
++
+*Yes:* "Cache created." *No:* "Cache has been created."
+* Avoid using the word "please".
+* Ensure all icons and buttons with action have tooltip/hover help.
```

Comment:
For accessibility reasons, we have on PatternFly to add a tooltip just for semantic icons (not decorative icons) that don't have labels (e.g., a trash can icon would have a tooltip reading "Delete"). A tooltip isn't needed for buttons or anything else with labels. So we could say something like, 

_For semantic icons without labels, add a label inside a tooltip that appears on hover. For more information on accessible icons, see [PatternFly's accessibility guide.](https://www.patternfly.org/v4/get-started/accessibility-guide)_